Configuration/Settings.Jonnitto.yaml

```diff
@@ -37,4 +37,4 @@ Jonnitto:
       showRelated: false
 
       # Whether the video plays inline on supported mobile devices. Note that if set to false it does not imply that the video will always be played in fullscreen.
-      playsinline: true
```

Comment:
Why do you want to change this? This would be a breaking change and in other packages it is also used (e.g. [PrettyEmbedVideoPlatforms](https://github.com/jonnitto/Jonnitto.PrettyEmbedVideoPlatforms/blob/master/Configuration/Settings.Jonnitto.yaml#L20))

Reply:
I wanted to change it to the camel case spelling. Because you have this in other places too. But i didnt know, that this would be a breaking change. I will reverse it again!